Management/src/ActuatorWeb/README.md

```diff
@@ -193,6 +193,8 @@ If you wish to collect and view application metrics, the [Metrics Registrar](htt
 1. `cf register-metrics-endpoint actuator-web-management-sample /actuator/prometheus --internal-port 8090`
 1. `cf register-metrics-endpoint actuator-api-management-sample /actuator/prometheus --internal-port 8091`
 1. [Add your own metric charts](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform-services/app-metrics-for-tanzu/2-2/app-metrics/using.html#custom-metrics)
+   1. Use an included .http file to send a request to the Prometheus to see what metrics are available
```

Comment:
Should be "to Prometheus" or "to the Prometheus server".

Reply:
Neither actually, updated to indicate that I meant the Prometheus _endpoint_